docs/getting-started/cli.md

```diff
@@ -72,6 +72,13 @@ To export the results to a JSON file:
 semgrep ci --json --output=semgrep.json
 ```
 
+To view your results in multiple formats, combine the appropriate flags:
+
+```console
+# obtain results as text output and as JSON, SARIF
+semgrep ci --json --text --output=semgrep.sarif --output=semgrep.json
```

Comment:
Pretty sure we have to use the new commands from Semgrep 1.70.0? 
```
semgrep ci --json-output=out.json --gitlab-sast-output=gitlab-sast.json
```

You can see the new commands: https://github.com/semgrep/semgrep-docs/pull/1504/files

You can confirm with Andre :) 
```

Reply:
Thank you for catching this! (Good news is everything is backwards compatible!)